backend/src/test/java/com/woowacourse/friendogly/club/service/ClubServiceTest.java

```diff
@@ -33,31 +33,32 @@ public abstract class ClubServiceTest extends ServiceTest {
             .sizeType(SizeType.SMALL)
             .build();
 
-    private final Club club = Club.create(
-            "강아지 산책시키실 분 모아요.",
-            "매주 주말에 정기적으로 산책 모임하실분만",
-            address,
-            5,
-            member,
-            allowedGenders,
-            allowedSizes,
-            "https://image.com");
-
-    protected Club saveNewClub() {
+    protected Club getSavedClub(Set<Gender> genders, Set<SizeType> sizes) {
```

Comment:
```suggestion
    protected Club createClub(Set<Gender> genders, Set<SizeType> sizes) {
```

메서드명은 get으로 되어 있지만 내부적으로는 생성도 하고 있네요.
create라는 이름은 어떨까요?